api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -103,6 +103,17 @@ message GrpcJsonTranscoder {
     bool reject_binding_body_field_collisions = 3;
   }
 
+  message ReflectionConfig {
+    // Supplies the cluster name of the gRPC cluster to use for reflection.
+    // This will typically be the same gRPC cluster that Envoy is forwarding
+    // gRPC traffic to, but it does not need to be.
+    string cluster_name = 1;
+
+    // The amount of time in ms that Envoy should wait for gRPC reflection
+    // requests to return.
+    uint64 request_timeout = 2;
```

Comment:
Could this be of type `google.protobuf.Duration`?
What is the default value?

Reply:
Yes, I can make it  a Duration. I'll document the default value regardless.

Reply:
How about adding `config.core.v3.RetryPolicy retry_policy`.  Since network failures are so bad,  it is better to have retry.  Can add it in a later pr.

Reply:
For `request_timeout`,  you can do

```
google.protobuf.Duration timeout = 3 [(validate.rules).duration = {
    required: true
    gte {}
  }];
```

Reply:
It's a fair point. I'll add it in this PR if it's easy.